src/GraphQL.SystemReactive/SubscriptionExecutionStrategy.cs

```diff
@@ -40,22 +40,25 @@ public override async Task<ExecutionResult> ExecuteAsync(ExecutionContext contex
                 if (!(node.FieldDefinition is EventStreamFieldType))
                     continue;
 
-                streams[node.Name] = await ResolveEventStreamAsync(context, node).ConfigureAwait(false);
+                var stream = await ResolveEventStreamAsync(context, node).ConfigureAwait(false);
+
+                if (stream != null)
```

Comment:
There was a bug in case of `null` value?

Reply:
No idea.  It was a long time ago.